### DISCUSSION TITLE
What changed?

### DISCUSSION THREAD

**Discussion by Mechanized82** [dev7355608/perfect-vision#290] [General]
Hi!  I'm having an issue figuring out what changed with some of the recent updates...

I have a scene that worked as I needed previously...  0 darkness gave full vision, 1 (100%) darkness resulted in pitch black for characters with basic vision.  

Currently, with global illumination off, the characters see everything as partially shaded during the day, and pitch black at 100% darkness... Or, I can turn on global illumination, and they see properly during the day but then 100% darkness still gives some minor amount of vision.  

Is there a new combination of settings I need in order to get this back the way it was?  I get that I can turn on/off the global illumination, but when doing a smooth transition from day>night or night>day it isn't very smooth anymore since both on/off global illumination settings have their limitations.  

I'm having trouble understanding what global illumination solves... isn't that what the 0-1 darkness slider did essentially?   

Edit: I believe adjusting the activation range fixes this... but it still remains visually unappealing, giving an abrupt change to pitch black, when doing night/day transitions.

**Comment by dev7355608** [comment]
PV removes the *Global Illumination Threshold* setting from the Scene config, but you find it under another name in the Global Illumination config: *Darkness Activation Range*, which you find in the normal light source config as well. It's the same scene setting under the hood though.

> I believe adjusting the activation range fixes this... but it still remains visually unappealing, giving an abrupt change to pitch black, when doing night/day transitions.

The transition was always abrupt (pre v10), because either the global light is on or off and so either allows vision or not. The only thing that might have changed is that the transition of the saturation of darkvision isn't smooth anymore. Are you referring to this also? So it might instantly switch from grayscale to colors once the global light is turned on. PV doesn't have any influence on that though: the system defines whether saturation of darkvision is linked to the darkness level (by default it isn't linked, so the saturation doesn't smoothly increase as the darkness level is decreased). You should get the same results whether PV is turned on or off in this regard; if not let me know.
